Modules/SaltStackConfig/SaltStackConfig.Format.ps1xml

```diff
@@ -24,13 +24,13 @@
                     <TableRowEntry>
                         <TableColumnItems>
                             <TableColumnItem>
-                                <PropertyName>SscServer</PropertyName>
+                                <PropertyName>Server Name</PropertyName>
```

Comment:
I would advice against property names with spaces, because to be able to use them you have to access them with quotes like this $connection.'Server Name'. In the vSphere module we use "Name" and "User". You could use the same property names for consistency if you want. If you prefer the ones that you have selected you could make them "ServerName" and "ConnectedAs".

Reply:
Thanks for the suggestions.  I've made this change and also adjusted the column widths so that they align with other PowerCLI cmdlets.  They'll appear in the next commit.  Marking as resolved.